### PR TITLE
LPS-122660 Flip button is enabled when there is no content categorized by the categories of the Audit Graph

### PR DESCRIPTION
**Pull request by darquesdev**
**Motivation**
The *flip axes* button must be enabled only if two vocabularies are being *represented* in the graph. 

**NOTE**: A vocabulary will be represented in the graph if these conditions are met:
- the vocabulary is selected in the In Use box in the Configuration panel
- the vocabulary has categories in it
- these categories are being used in some of hte web content that meet the current search & filters

**Approach**
Instead of taking the number of vocabularies from the configuration, these are obtained from the ones used to populate the graph

**How to test it**
There are integration tests and the steps described in the issue could be followed as well

### CODE REVIEW

**Comment by darquesdev** [comment]
ci:test:sf

**Comment by darquesdev** [comment]
ci:test:relevant

File: modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/display/context/ContentDashboardAdminDisplayContext.java [hunk 1]
```diff
@@ -394,11 +394,21 @@ public long getUserId() {
 	}
 
 	public boolean isSwapConfigurationEnabled() {
-		if (_assetVocabularies.size() == 2) {
-			return true;
+		if (_swapConfigurationEnabled != null) {
```
Comment (cgoncas) [inline modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/display/context/ContentDashboardAdminDisplayContext.java:397]: Why not return swapConfigurationEnabled directly?

Comment (cgoncas) [reply modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/display/context/ContentDashboardAdminDisplayContext.java:397]: Is this a mutable value? Could it not being initialized in the constructor?

**Review by cgoncas — APPROVED** [review]
LGTM! I have dome doubts about the initialization of the swapConfigurationEnabled variable. But I will be fine with the decisión you take.

File: modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/display/context/ContentDashboardAdminDisplayContext.java (hunk 1 above)
Comment (darquesdev) [reply modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/display/context/ContentDashboardAdminDisplayContext.java:397]: that's true, I'm following the same approach as with the rest of properties. But I really like this way you are proposing

Comment (darquesdev) [reply modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/display/context/ContentDashboardAdminDisplayContext.java:397]: but i'm going to keep it as it is, other way I may feel uncomfortable having some  properties being initialized in the constructor and others in its getter method. But this is something we can consider. I agree with you in setting variable as final and initializing them in the constructor.

**Comment by darquesdev** [comment]
ci:forward

**Pull request closed without merging**